apps/main/src/modules/wallet/assets/MyLiquidity/LiquidityDetailMobileActions.tsx

```diff
@@ -1,12 +1,21 @@
 import { Button, Grid } from "@galacticcouncil/ui/components"
+import { Link } from "@tanstack/react-router"
+import { FC } from "react"
 import { useTranslation } from "react-i18next"
 
-export const LiquidityDetailMobileActions = () => {
+type Props = {
+  readonly assetId: string
+}
+
+export const LiquidityDetailMobileActions: FC<Props> = ({ assetId }) => {
   const { t } = useTranslation("wallet")
 
   return (
     <Grid columnGap={8} sx={{ gridTemplateColumns: "1fr 1fr" }}>
-      <Button size="large">{t("myLiquidity.actions.poolDetails")}</Button>
+      <Link to={"/liquidity/$id"} params={{ id: assetId }}>
+        <Button size="large">{t("myLiquidity.actions.poolDetails")}</Button>
+      </Link>
```

Comment:
Button inside Link is not really valid HTML markup (2 nested interactive elements)

You can use asChild prop on Button to render Link with Button styles

```suggestion
      <Button size="large" asChild>
        <Link to={"/liquidity/$id"} params={{ id: assetId }}>
          {t("myLiquidity.actions.poolDetails")}
        </Link>
      </Button>
```

Reply:
Does not play with well `iconStart` and `iconEnd` so I reworked the composition too.

Reply:
I dont think that was necessary, if you need asChild prop and icons, you could still add icons as children, for all other cases, iconStart, iconEnd props would work.